source/_components/hangouts.markdown

```diff
@@ -22,11 +22,13 @@ Configure the integration:
 * Enter your **Google Mail Address** and **Password**
 * If you secured your account with 2-factor authentication you will be asked for a 2-factor authentication token.
 
-**BEST PRACTICE:** You can't write messages to yourself or get notifications in a group, if "you" write the message. The best way is to create a new Google Hangouts account for this integration.
-
-**IMPORTANT:** If you secured your account with 2-factor authentication: Only verification by app or SMS are supported. There is no support for verification by prompt on your phone.
-
-**IMPORTANT 2:** If you are sure your email and password is correct, but the component says the login is invalid. Wait a few hours and try it again, it might be, that google asks for a captcha which we can't support. Google does not provide official support for using bots with Google Hangouts, that's why we have to work around this.
+<p class='note'>
+<ul>
```

Comment:
[It is not working as intended, apparently](https://deploy-preview-6428--home-assistant-docs.netlify.com/components/hangouts). Text inside HTML tags may be written in Markdown too. [HTML usage is not recommended](https://developers.home-assistant.io/docs/en/documentation_create_page.html#html)

Reply:
I wonder why though. Github preview parses it as HTML. Indentation?

Reply:
The GitHub preview doesn't support everything.

Reply:
@fabaff Could you tell then how this feature works? Does the Home Assistant website render only particular HTML code, leaving the rest raw?

Reply:
I read in the Markdown specs that Markdown inside html blocks like p won't be rendered.. interesting. Gonna fix.